### ISSUE TITLE
Bug: I see a glitch, when i click, hover on the taskbar

### ISSUE THREAD

**Issue by anburocky3** [jmalarcon/Win11ClockToggler#4]
It is happening for a second, but it is visible while recording.
I have 4k monitor, i always increase my resolution to show the text clearer. Even in that, i see this problem arises.

**Comment by jmalarcon** [comment]
Hi: this is because when you do that, Windows automatically shows everything again. The program monitors this action and hides it as soon as it cans, but it could be shown for a split of a second.

Take into account that the program uses some "hacks" to hide the elements, since the operating system has no support for it, and it's not 100% perfect.

Disable notifications while recording, as I suggest in the readme file of the repo.

HTH.

**Comment by anburocky3** [comment]
Mm can understand, while i do minimize my taskbar and enabled, hover on show. while hovering the taskbar, it is visible. 🥲 

Thank you for your wonderful software. It is wonderful.

**Comment by jmalarcon** [comment]
Ah, OK, I see. If you have your taskbar automatically hidden (you should have told me that from the beginning!) the program can see it and it can't automatically hide the zones again until it's completely shown 🤔

I don't know if I could do anything with that, but I'll investigate it further. Thanks!

**Comment by anburocky3** [comment]
> Ah, OK, I see. If you have your taskbar automatically hidden (you should have told me that from the beginning!) 

No. even if i enable taskbar at all times, while clicking on taskbar, it changes. but i can understand the technical difficulties you are facing. Hope, we will have an workaround to fix it. Thanks.

**Comment by jmalarcon** [comment]
HI again:

That's really strange. There's no reason for it to be shown if you hover the mouse or click over the taskbar or notification area.

You must take into account that Windows shows everything again (date/time and notification icons) if there's any **change** in that area. Therefore if you have any notification icon that is changing all the time, a new notification arrives, etc... everything will be shown again and my app will try to hide it as soon as possible but still there's a chance to be seen for a small twitch.

Do you have any program that changes an icon on the notification area? it must be something like that. I'm sure is something relates to your own computer and the software you are using...
